### ISSUE TITLE
aws_route53_record not respecting zone_id in (...)

### ISSUE THREAD

**Issue by tinder-tder** [turbot/steampipe-plugin-aws#898]
**Describe the bug**
When querying the aws_route53_record table using `... where zone_id  in ('aaaa')` works but with `('aaaa','bbbb')` it gives no results when both zone ids have records.

**Steampipe version (`steampipe -v`)**
v0.12.2

**Plugin version (`steampipe plugin list`)**
v0.48.0

**To reproduce**
```
> select count(name) from dev1.aws_route53_record where zone_id in ('AAAAA')
+-------+
| count |
+-------+
| 59    |
+-------+
> select count(name) from dev1.aws_route53_record where zone_id in ('BBBBB')
+-------+
| count |
+-------+
| 124   |
+-------+
> select count(name) from dev1.aws_route53_record where zone_id in ('AAAAA','BBBBB')
+-------+
| count |
+-------+
| 0     |
+-------+
```

**Expected behavior**
records from all specified zone_ids to be available

**Comment by rajlearner17** [comment]
@tinder-tder Appreciate continuous input. This is an interesting issue, we are looking into it.

**Comment by kaidaguerre** [comment]
@tinder-tder many thanks for raising this issue. I've found a bug/edge case in the way the plugin sdk handles `in` queries: https://github.com/turbot/steampipe-plugin-sdk/issues/275

I'll update here as we progress with a fix for that.

**Comment by tinder-tder** [comment]
Thanks @rajlearner17 @kaidaguerre for the quick response!

**Comment by misraved** [comment]
@tinder-tder We have released the [AWS plugin (0.52.0)](https://github.com/turbot/steampipe-plugin-aws/blob/main/CHANGELOG.md#v0520-2022-03-23) with Steampipe Plugin SDK v2.1.1.

This should solve the above issue. Please let us know in case you still run into the same issue 👍

**Comment by tinder-tder** [comment]
@misraved Thanks, sorry I was OOO. The new versions work correctly
